fuzzers/Makefile

```diff
@@ -19,161 +19,169 @@ clean_locks:
+$(eval $(call fuzzer,012-clb-n5ffmux,005-tilegrid,N))
+$(eval $(call fuzzer,013-clb-ncy0,005-tilegrid,N))
+$(eval $(call fuzzer,014-clb-ffsrcemux,005-tilegrid,N))
+$(eval $(call fuzzer,015-clb-nffmux,005-tilegrid,N))
+$(eval $(call fuzzer,016-clb-noutmux,005-tilegrid,N))
+$(eval $(call fuzzer,017-clb-precyinit,005-tilegrid,N))
+$(eval $(call fuzzer,018-clb-ram,005-tilegrid,N))
+$(eval $(call fuzzer,019-clb-ndi1mux,005-tilegrid,N))
+$(eval $(call fuzzer,025-bram-config,005-tilegrid,N))
+$(eval $(call fuzzer,026-bram-data,005-tilegrid,N))
+$(eval $(call fuzzer,027-bram36-config,005-tilegrid,N))
+$(eval $(call fuzzer,028-fifo-config,005-tilegrid,N))
+$(eval $(call fuzzer,029-bram-fifo-config,005-tilegrid,N))
+$(eval $(call fuzzer,030-iob,005-tilegrid,N))
+$(eval $(call fuzzer,032-cmt-pll,005-tilegrid,N))
+$(eval $(call fuzzer,034-cmt-pll-pips,005-tilegrid 071-ppips,N))
+$(eval $(call fuzzer,035-iob-ilogic,005-tilegrid,N))
+$(eval $(call fuzzer,035a-iob-idelay,005-tilegrid,N))
+$(eval $(call fuzzer,036-iob-ologic,005-tilegrid,N))
+$(eval $(call fuzzer,037-iob-pips,005-tilegrid,N))
+$(eval $(call fuzzer,038-cfg,005-tilegrid,N))
+$(eval $(call fuzzer,039-hclk-config,005-tilegrid,N))
+$(eval $(call fuzzer,040-clk-hrow-config,005-tilegrid,N))
+$(eval $(call fuzzer,041-clk-hrow-pips,005-tilegrid,N))
+$(eval $(call fuzzer,042-clk-bufg-config,005-tilegrid,N))
+$(eval $(call fuzzer,043-clk-rebuf-pips,005-tilegrid,N))
+$(eval $(call fuzzer,044-clk-bufg-pips,046-clk-bufg-muxed-pips,N))
+$(eval $(call fuzzer,045-hclk-cmt-pips,005-tilegrid,N))
+$(eval $(call fuzzer,046-clk-bufg-muxed-pips,005-tilegrid,N))
+$(eval $(call fuzzer,047-hclk-ioi-pips,005-tilegrid,N))
+$(eval $(call fuzzer,047a-hclk-idelayctrl-pips,047-hclk-ioi-pips,N))
+$(eval $(call fuzzer,048-int-piplist,005-tilegrid,N))
+$(eval $(call fuzzer,049-int-imux-gfan,048-int-piplist,N))
+$(eval $(call fuzzer,050-pip-seed,048-int-piplist,N))
+$(eval $(call fuzzer,051-pip-imuxlout-bypalts,048-int-piplist,N))
+$(eval $(call fuzzer,052-pip-clkin,048-int-piplist,N))
+$(eval $(call fuzzer,053-pip-ctrlin,048-int-piplist,N))
+$(eval $(call fuzzer,054-pip-fan-alt,048-int-piplist,N))
+$(eval $(call fuzzer,055-pip-gnd,048-int-piplist,N))
+$(eval $(call fuzzer,056-pip-rem,049-int-imux-gfan 050-pip-seed 051-pip-imuxlout-bypalts 052-pip-clkin 053-pip-ctrlin 054-pip-fan-alt 055-pip-gnd 059-pip-byp-bounce,N))
+$(eval $(call fuzzer,057-pip-bi,056-pip-rem,N))
 ifneq ($(QUICK),Y)
-$(eval $(call fuzzer,058-pip-hclk,005-tilegrid))
-$(eval $(call fuzzer,059-pip-byp-bounce,048-int-piplist))
-$(eval $(call fuzzer,060-bram-cascades,005-tilegrid))
-$(eval $(call fuzzer,071-ppips,057-pip-bi 058-pip-hclk 060-bram-cascades))
+$(eval $(call fuzzer,058-pip-hclk,005-tilegrid,N))
+$(eval $(call fuzzer,059-pip-byp-bounce,048-int-piplist,N))
+$(eval $(call fuzzer,060-bram-cascades,005-tilegrid,N))
+$(eval $(call fuzzer,071-ppips,057-pip-bi 058-pip-hclk 060-bram-cascades,N))
 ifneq ($(BITONLY),Y)
```

Comment:
Should the following still be `Y`?

Reply:
Which line are you refering too?  I don't follow what you are referring too.

Reply:
The last value in the `$(eval)` is the verbose setting, right? Should it still be verbose for these fuzzers?
```
$(eval $(call fuzzer,072-ordered_wires,,Y))
$(eval $(call fuzzer,073-get_counts,))	$(eval $(call fuzzer,073-get_counts,,Y))
$(eval $(call fuzzer,074-dump_all,005-tilegrid 072-ordered_wires))	$(eval $(call fuzzer,074-dump_all,005-tilegrid 072-ordered_wires,Y))
$(eval $(call fuzzer,075-pins,))	$(eval $(call fuzzer,075-pins,,Y))
```

Reply:
Fixed.